### ISSUE TITLE
Possible exception when receiving message from another player.

### ISSUE THREAD

**Issue by Measurity** [SubnauticaNitrox/Nitrox#876]
Assumption: chatlog isn't loaded yet for receiving player

**Error log (I'm running 2 instances of Subnautica at once so it looks a bit weird)**
```
[21:56:50 INFO]: Trying to load scene: Assets/chatlog.unity
[21:56:50 DEBUG]: Scene 'chatlog' loaded as Additive
[21:56:51 ERROR]: Error processing packet: NitroxModel.Packets.ChatMessage
System.NullReferenceException: Object reference not set to an instance of an object
  at NitroxClient.MonoBehaviours.Gui.Chat.PlayerChatLog.BuildChatText () [0x00042] in <27ea193084534899831c4836781ed9a6>:0 
  at NitroxClient.MonoBehaviours.Gui.Chat.PlayerChatLog.WriteEntry (NitroxClient.GameLogic.ChatUI.ChatLogEntry chatLogEntry) [0x00038] in <27ea193084534899831c4836781ed9a6>:0 
  at NitroxClient.GameLogic.ChatUI.PlayerChat.WriteChatLogEntry (NitroxClient.GameLogic.ChatUI.ChatLogEntry chatLogEntry) [0x00001] in <27ea193084534899831c4836781ed9a6>:0 
  at NitroxClient.Communication.Packets.Processors.ChatMessageProcessor.LogClientMessage (NitroxModel.Packets.ChatMessage message) [0x00047] in <27ea193084534899831c4836781ed9a6>:0 
  at NitroxClient.Communication.Packets.Processors.ChatMessageProcessor.Process (NitroxModel.Packets.ChatMessage message) [0x00016] in <27ea193084534899831c4836781ed9a6>:0 
  at NitroxClient.Communication.Packets.Processors.Abstract.ClientPacketProcessor`1[T].ProcessPacket (NitroxModel.Packets.Packet packet, NitroxModel.Packets.Processors.Abstract.IProcessorContext context) [0x00001] in <27ea193084534899831c4836781ed9a6>:0 
  at NitroxClient.MonoBehaviours.Multiplayer.ProcessPackets () [0x00055] in <27ea193084534899831c4836781ed9a6>:0 
[21:56:51 INFO]: Trying to load scene: Assets/chatlog.unity
[21:56:51 DEBUG]: Scene 'chatlog' loaded as Additive
```

**Comment by Measurity** [comment]
This issue will be fixed once I'm done creating a new UI api which we will use for custom UIs in Nitrox instead of the chatlog asset bundle.

**Comment by Measurity** [comment]
This has been fixed as of #997 ([see PlayerChat.cs changes](https://github.com/SubnauticaNitrox/Nitrox/pull/997/files#diff-33833de2293bb6e697d5f295f31dd9cbR31-R42))
